packages/app-data/src/api/fetchDocFromAppData.ts

```
@@ -0,0 +1,53 @@
+export async function fetchDocFromAppDataHexLegacy(
+  appDataHex: string,
+  ipfsUri?: string
+): Promise<void | AnyAppDataDocVersion> {
+  return _fetchDocFromCidAux(appDataHexToCidLegacy, appDataHex, ipfsUri)
+}
+
+export async function _fetchDocFromCidAux(
+  hexToCid: (appDataHex: string) => Promise<string>,
+  appDataHex: string,
+  ipfsUri?: string
+): Promise<void | AnyAppDataDocVersion> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)